src/constants/locConstants.ts

```diff
@@ -1329,6 +1329,11 @@ export class PublishProject {
     public static PublishTargetNewAzureServer = l10n.t("New Azure SQL logical server (Preview)");
     public static GenerateScript = l10n.t("Generate Script");
     public static Publish = l10n.t("Publish");
+    public static AdvancedOptions = l10n.t("Advanced");
+    public static AdvancedPublishSettings = l10n.t("Advanced Publish Options");
+    public static GeneralOptions = l10n.t("General Options");
+    public static IgnoreOptions = l10n.t("Ignore Options");
+    public static ExcludeObjectTypes = l10n.t("Exclude Object Types");
```

Comment:
Are these getting used anywhere?  They look like they're UI strings, and I see you already have them in `reactviews/common/locConstants.ts`.